test/data/buyAndBurn.ts

```diff
@@ -1385,6 +1385,221 @@ describe("BuyAndBurnSwap", () => {
       // With greedy strategy, we should have burned significant DLP
       totalDLPBurned.should.be.gt(0);
     });
+
+    it("should handle iterative swaps using spareIn with pool state changes", async () => {
```

Comment:
@volod-vana Is it the test for multiple calls of `swapAndAddLiquidity` to simulate the cron job? If it is, I don't think it implemented what we discussed.